overlay/olsettings.cpp

```diff
@@ -58,17 +58,17 @@ static std::vector<std::string> regReadMultiString(HKEY key,
 {
 	LONG err = 0;
 	std::vector<std::string> out;
-	char *buf = NULL;
+	char *buf = nullptr;
 	HKEY subKeyHandle = 0;
 
-	err = RegOpenKeyExA(key, subKey.c_str(), NULL, KEY_READ, &subKeyHandle);
+	err = RegOpenKeyExA(key, subKey.c_str(), nullptr, KEY_READ, &subKeyHandle);
```

Comment:
So if I understand this correctly this NULL should have been a 0 in the first place?
https://docs.microsoft.com/en-us/windows/win32/api/winreg/nf-winreg-regopenkeyexa

Sorry if that's a dumb question but I have no win32 experience and no Windows compiler at hand.

Reply:
I think so, yes :thinking: